diff --git a/core/lib/Drupal/Core/Database/Driver/sqlite/Install/Tasks.php b/core/lib/Drupal/Core/Database/Driver/sqlite/Install/Tasks.php
index 81bdea39..847ee206 100644
--- a/core/lib/Drupal/Core/Database/Driver/sqlite/Install/Tasks.php
+++ b/core/lib/Drupal/Core/Database/Driver/sqlite/Install/Tasks.php
@@ -18,7 +18,7 @@ class Tasks extends InstallTasks {
    * Use to build sqlite library with json1 option for JSON datatype support.
    * @see https://www.sqlite.org/json1.html
    */
-  const SQLITE_MINIMUM_VERSION = '3.26';
+  const SQLITE_MINIMUM_VERSION = '3.16';
 
   /**
    * {@inheritdoc}
